### PR TITLE
Updating adviser promotion around site

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/cVdo6R35/3988-promote-benefits-and-process-of-the-adviser-service-across-the-site

### Context

The TTA service is experiencing a reduction in sign-ups compared with last year, which in turn was worse than the year before.

We should review where we currently promote the adviser service around the site.

### Changes proposed in this pull request

This PR focuses on:

- how we promote the benefits of the adviser service
- how we promote how the adviser service works

Most of these changes are looking at our promo modules/footers, so there's obviously only so much space for the copy. It's a bit of a balancing act between explaining exactly what advisers can do, and how it works. I've spoken with @philspray about this and we agreed to prioritise the **how** of the adviser service in these instances. This makes more sense as there are far more benefits of the adviser service than there is space to list for each user group, and not understanding how the service works (i.e. that it is up to the candidate how often they use the service and how, the fact that advisers are experienced teachers) is more likely to be more of a stumbling block to them signing up.

Naeem also sent through some of the adviser service key messages, which I've tried to incorporate in some places (e.g. 'Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience', 'Do not let doubts of changing career hold you back – our experienced teaching advisers can support you through the switch.')

### CODE REVIEW

File: app/views/content/non-uk-teachers/promos/_adviser-promo-teach-in-england.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <% promo.right_section(
     heading: "Get free one-to-one support"
   ) do %>
-      <p>An adviser can answer all your questions about what it's like teaching in England and what you need to get into an English classroom.</p>
+      <p>An adviser can answer all your questions about what it's like teaching in England and how to get into an English classroom. Chat through text, email or phone, as little or as often as you need.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/promos/_adviser-promo-teach-in-england.html.erb:6]: @EllieNodder Minor comment (as this is already in prod) - 'how to get into an English classroom' sounds a bit odd and colloquial (through the door is surely the answer!) - worth rewording slightly?

File: app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
   <% promo.left_section(classes: %w[tta-background]) %>
   <% promo.right_section(heading: "Get free one-to-one support") do %>
-      <p>An adviser can talk you through the support available if you're changing career and answer all your questions about getting into teaching.</p>
+      <p>Do not let doubts of changing career hold you back – our experienced teaching advisers can support you through the switch. Chat to an adviser through email, text, or phone.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb:4]:
```suggestion
      <p>Don't let doubts of changing career hold you back – our experienced teaching advisers can support you through the switch. Chat to an adviser through email, text, or phone.</p>
```
Not sure about this @EllieNodder - I like the second bit about 'our experienced teaching advisers.....' but wonder about the doubts bit, as there will be a lot of considerations (financial, family, etc.)?

Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb:4]: this was one that I changed to match up with the key messages Naeem sent across but happy to tweak!

File: app/views/content/blog/from-the-army-to-teacher-training.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ I was invited to all 3 interviews and subsequently gained a place. I’m lucky a
 
 I can’t speak highly enough of Elaine and the assistance given by the Get Into Teaching service. I'm looking forward to starting my primary PGCE with PE specialisation at Sheffield Hallam University.
 
-If you're interested in changing your career like David, [find out more about a career change into teaching](/train-to-be-a-teacher/if-you-want-to-change-career).
+If you're interested in changing your career like David, [find out more about a career change into teaching](/train-to-be-a-teacher/if-you-want-to-change-career). Or get free, one-to-one support from a [teacher training adviser](/teacher-training-advisers) with years of teaching experience by email, text or phone, as often or as little as you need.
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/blog/from-the-army-to-teacher-training.md:37]: have mentioned this to Andy/Naeem - shall bring up later in our catch up

File: app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb (hunk 2 above)
Comment (gemmadallmandfe) [reply app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb]: Happy for you to take a call on this :-) Keisha might have some useful insight?

**Review by gemmadallmandfe — DISMISSED** [review]
Looks good to me!

File: app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md [hunk 4]
```diff
@@ -51,7 +51,9 @@ Whilst school experience isn’t compulsory, it’s a good idea to visit a schoo
 
 ## 3.	Get a teacher training adviser
 
-Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application.
+All our teacher training advisers are experienced teachers who can provide you with additional support when preparing and applying for teacher training. You could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application.
+
+Chat by phone, text or email, as little or as often as you need.
 
 [Get a teacher training adviser](https://adviser-getintoteaching.education.gov.uk/).
```
Comment (gemmadallmandfe) [inline app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md:58]: Do we want to take people to the TTA page here or deliberately to the funnel as they could be ETA candidates @EllieNodder ?

File: app/components/footer/talk_to_us_component.html.erb [hunk 5]
```diff
@@ -22,7 +22,7 @@
           <div>
             <h3 class="heading-s">Get free one-to-one support</h3>
             <p>
-              Our advisers provide support wherever you are in your journey into teaching. Whether it's just an idea, you're ready to apply or you're looking to get back into the classroom, you could get personalised help and advice.
+              Whether it's just an idea or you're ready to apply, you could get personalised support from an adviser with years of teaching experience. Chat to them through phone, email or text as little or as often as you need.
```
Comment (gemmadallmandfe) [inline app/components/footer/talk_to_us_component.html.erb:25]:
```suggestion
              Whether it's just an idea or you're ready to apply, you could get personalised support from an adviser with years of teaching experience. Chat to them by phone, email or text as little or as often as you need.
```
change this one too for consistency @EllieNodder ?

File: app/views/content/home/_calls-to-action.html.erb [hunk 6]
```diff
@@ -2,7 +2,7 @@
   <%= render CallsToAction::HomepageComponent.new(
     icon: "icon-person",
     title: "Get one-to-one support",
-    text: "Talk to an adviser with years of teaching experience to find out if teaching is for you and how to take your next steps.",
+    text: "Talk to an adviser with years of teaching experience to find out if teaching is for you. Chat through phone, text or email.",
```
Comment (gemmadallmandfe) [inline app/views/content/home/_calls-to-action.html.erb:5]:
```suggestion
    text: "Talk to an adviser with years of teaching experience to find out if teaching is for you. Chat by phone, text or email.",
```
should we change this to by for consistency @EllieNodder ?

File: app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md (hunk 4 above)
Comment (EllieNodder) [reply app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md:58]: hmm probably TTA page actually, if it's a blog about applying

File: app/views/content/non-uk-teachers/promos/_adviser-promo-ukraine.html.erb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(classes: %w[tta-background]) %>
+  <% promo.right_section(
+    heading: "Get free one-to-one support"
+  ) do %>
+      <p>An adviser with years of teaching experience can answer all your questions about becoming a teacher England. Chat by phone, text or email, as little or as often as you need.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/promos/_adviser-promo-ukraine.html.erb:6]:
```suggestion
      <p>An adviser with years of teaching experience can answer all your questions about becoming a teacher in England. Chat by phone, text or email, as little or as often as you need.</p>
```

File: app/views/content/non-uk-teachers/promos/_adviser-promo-returners.html.erb [hunk 8]
```diff
@@ -3,7 +3,7 @@
   <% promo.right_section(
     heading: "Get free one-to-one support"
   ) do %>
-      <p>Talk to an adviser about the best way to get back to teaching in an English classroom with your international skills and experience.</p>
+      <p>Talk to an adviser about the best way to get back to teaching in an English classroom with your international skills and experience. Chat through phone, text or email, as little or as often as you need.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/promos/_adviser-promo-returners.html.erb:6]:
```suggestion
      <p>Talk to an adviser about the best way to get back to teaching in an English classroom with your international skills and experience. Chat by phone, text or email, as little or as often as you need.</p>
```
change for consistency @EllieNodder ?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**